builders/tools/explorers/verify-contract.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Verify Smart Contracts
+description: Learn about how to verify smart contracts on Moonbeam networks using one of the available block explorers. 
+
+---
+# Verify Smart Contracts
+
+![Explorer Banner](/images/builders/tools/explorers/overview/explorers-banner.png)
+
+## Introduction {: #introduction } 
+
+Verifying smart contracts on a block explorer is great way of improving the transparency and security of deployed smart smarts on Moonbeam. Users can directly view the source code on verified smart contracts on the block explorer, and can also directly interact with the contract's public methods through the block explorer's interface. 
```

Comment:
also moonscan seems to be the only explorer you can interact with the methods... so I would maybe say "on some explorers you can also directly interact..." or something along those lines

Reply:
BlockScout supports reading and writing to verified contract as well (it creates two new tabs), but I'll add something to that effect in case for Subscan etc.